config/production.yaml

```diff
@@ -0,0 +1,18 @@
+---
+PLAINTEXT_VARIABLES:
```

Comment:
This is a bit of difference between the variables specified for prod and dev, is that intended? I'm looking especially at dev missing `KINESIS_BATCH_SIZE` and some of the `_DATE` missing from prod.

Reply:
Yeah this is on purpose. The dev run doesn't use the cache or Kinesis (it sets `IGNORE_CACHE` and `IGNORE_KINESIS` to true) so it doesn't need the Kinesis batch size/stream name or the S3 bucket/resource. Instead of the cache it uses the `LAST_POLL_DATE` and `END_DATE` environment variables.